tests/ffi/tests.cc

```diff
@@ -872,7 +876,7 @@ extern "C" const char *cxx_run_test() noexcept {
   ASSERT(cstr == "foo");
   ASSERT(other_cstr == "test");
 
-  const char *utf8_literal = u8"Test string";
+  const char *utf8_literal = reinterpret_cast<const char*>(u8"Test string");
```

Comment:
Why is this change required?

Reply:
Because we need to use C++20 to compile Seastar, and this type of cast is not allowed in C++20